pkg/nodeconfig/payload/payload.go

```diff
@@ -188,13 +224,13 @@ func PopulateNetworkConfScript(clusterCIDR, hnsNetworkName, hnsPSModulePath, cni
 
 // generateNetworkConfigScript generates the contents of the .ps1 file responsible for CNI configuration
 func generateNetworkConfigScript(clusterCIDR, hnsNetworkName, hnsPSModulePath,
-	cniConfigPath string) (string, error) {
+	cniDir string) (string, error) {
 	networkConfScript := networkConfTemplate
 	for key, val := range map[string]string{
 		"HNS_NETWORK":          hnsNetworkName,
 		"SERVICE_NETWORK_CIDR": clusterCIDR,
 		"HNS_MODULE_PATH":      hnsPSModulePath,
-		"CNI_CONFIG_PATH":      cniConfigPath,
```

Comment:
Should we also pass in cniConfigDir to the method and create an entry for it here? Would help us avoid hardcoding `$cni_config_file` in the networkConf script

Reply:
+1